roberta/ev3.py

```diff
@@ -102,6 +102,19 @@ def makeMediumMotor(port, regulated, direction, side=None):
             m = None
         return m
 
+    @staticmethod
+    def makeOtherConsumer(port, regulated, direction, side=None):
+        try:
+            lp = ev3dev.LegoPort(port)
+            lp.mode = 'dc-motor'
+            time.sleep(0.5)
```

Comment:
please add a comment why that is needed, ideally with references if this is stated on some webpage.